android/KMEA/app/src/main/java/com/tavultesoft/kmea/KeyboardPickerActivity.java

```diff
@@ -107,7 +105,12 @@ public void onCreate(Bundle savedInstanceState) {
       kbInfo.put(KMManager.KMKey_CustomKeyboard, "N");
       kbInfo.put(KMManager.KMKey_Font, KMManager.KMDefault_KeyboardFont);
       keyboardsList.add(kbInfo);
-      saveKeyboardsList(context);
+
+      // We'd prefer not to overwrite a file if it exists
```

Comment:
Right. keyboardsList could be null because either
1. file doesn't exist
2. access error of an existing file

alas, I'm not sure how we could possibly recover from the 2nd scenario.